lib/auto.js

```diff
@@ -30,6 +30,15 @@ function findAutoVersionAsync(cwd) {
 		} catch (e) {
 			Error.throwIfNot(Error.ENOENT, e, 'Failed to read file: ' + versionFile);
 		}
+
+		// If we don't have a version string, try checking for an .nvmrc file
+		if (!versionString && !settings.disableNvmrc) {
+			versionFile = path.join(dir, '.nvmrc');
+			try {
+				versionString = fs.readFileSync(versionFile, 'utf8').trim();
+			} catch (_) { }
```

Comment:
I'd suggest the same error handling as on line 31 above.

Reply:
I think I actually misunderstood what the error handling above was doing. Now that I'd looked again, I get it. Adding now.